server/serverimpl_test.go

```diff
@@ -275,6 +360,10 @@ func TestDisconnectWSConnection(t *testing.T) {
 		_, _, err := conn.ReadMessage()
 		return err != nil
 	})
+
+	require.Equal(t, 1, len(logger.logs))
+	require.Contains(t, logger.logs[0], "Errorf: Cannot read a message from WebSocket: read tcp")
+	require.Contains(t, logger.logs[0], "use of closed network connection")
```

Comment:
Do we care about the log messages produced? It looks fragile to make the tests dependent on the exact formatting of the messages.

Reply:
Agree we should avoid fragility. The original motivation for this change was to deal with superfluous error logs produced by the server-side Disconnect() calls. Open to suggestions for a better criteria for to test for.

Reply:
Sounds like a good compromise. All set.